README.md

```diff
@@ -1870,6 +1870,26 @@ _You can enable the following settings in Xcode by running [this script](resourc
 
   </details>
 
+* <a id='doc-comments-before-attributes'></a>(<a href='#doc-comments-before-declarations'>link</a>) **Place doc comments for a declaration before any attributes.** [![SwiftFormat: docCommentsBeforeAttributes](https://img.shields.io/badge/SwiftFormat-docCommentsBeforeAttributes-7B0051.svg)](https://github.com/nicklockwood/SwiftFormat/blob/master/Rules.md#docCommentsBeforeAttributes)
```

Comment:
I like the placement of the rule below a similar rule 👍 
<img width="1176" alt="Screenshot 2024-07-26 at 4 44 50 PM" src="https://github.com/user-attachments/assets/57ff3db8-e5b5-4905-817f-e9963dc5bdcd">